docs/Reply.md

```diff
@@ -438,6 +499,6 @@ the socket has been destroyed.
 
 For more details, see:
 
-- https://github.com/fastify/fastify/issues/1864 for the discussion about this feature
-- https://promisesaplus.com/ for the definition of thenables
-- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then for the signature
+- <https://github.com/fastify/fastify/issues/1864> for the discussion about this feature
+- <https://promisesaplus.com/> for the definition of thenables
+- <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then> for the signature
```

Comment:
More seemingly random `<` and `>` additions.

Reply:
Its flagging bare URLs, maybe what we should do is:

```
- [discussion about this feature](https://github.com/fastify/fastify/issues/1864) 
- [definition of thenables](https://promisesaplus.com)
- [signature](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then)
```

WDYT?

Reply:
I think the author chose to use bare links instead of titled links for a reason. I just don't think this rule is needed.